EXE/flhook_plugins/FLSR-Crafting.cfg

```diff
@@ -893,3 +893,55 @@ ingredient = commodity_superconductors, 10
 ingredient = commodity_construction_machinery, 10
 cost = 0
 success_sound_nickname = Crafting_successful_equipment
+
+;; ----------------------------
+;; EQUIPMENT CRAFTS (ENGINES)
+;; ----------------------------
+
+;; - Outcast Transport Engine -
+;; 		Available on:
+;;				Everywhere
+
+[Outcast Transport Engine]
```

Comment:
Actually, the faction is called "Outcasts" and not "Outcast". Do we have "Outcast" in singular everywhere else in the game? Same for Corsairs. Do ships of the Corsairs say "Corsair" or "Corsairs"?